WatchDogCore/WatchDog/src/nl/tudelft/watchdog/core/logic/document/Document.java

```diff
@@ -75,12 +75,12 @@ public void setDocumentType(DocumentType type) {
 
 	/** Prepares this document to extract statistics out of it. */
 	public void prepareDocument() {
-		this.nameHash = WatchDogUtils.createFileNameHash(name);
+		this.nameHash = WatchDogUtilsBase.createFileNameHash(name);
```

Comment:
Do we have a WatchDogUtils class that is plugin-specific, too? Otherwise, I suggest simply WatchDogUtils